nornir/core/inventory.py

```diff
@@ -438,3 +438,19 @@ def children_of_group(self, group: Union[str, Group]) -> Set[Host]:
             if host.has_parent_group(group):
                 hosts.add(host)
         return hosts
+
+    def add_host(self, name: str, defaults: Optional[Defaults] = None,
```

Comment:
You are accepting `defaults` as an argument but you are not really using it :) In any case, it's not really necessary so removing it should be ok.